Gemfile

```diff
@@ -2,5 +2,6 @@
 
 source 'https://rubygems.org'
 
-gem 'fastlane', '~> 2.228'
+gem 'fastlane', '~> 2.227'
 gem 'fastlane-plugin-wpmreleasetoolkit', '~> 13.2'
+gem 'fluent-tools', '~> 0.1', git: 'https://github.com/Automattic/fluent-rust-tools.git', branch: 'iangmaia/ruby-gem-support'
```

Comment:
🎗️ As you mentioned in the PR description, reminder to update this once the gem is published before merging this PR